trade-finance-manager-api/src/v1/routes.js

```diff
@@ -67,6 +68,13 @@ authRouter.use('/', tasksRouter);
  */
 openRouter.route('/feedback').post(feedbackController.create);
 
+// token-validator
+authRouter.get('/validate-user-token', (_req, res) => res.status(200).send());
+
+openRouter.route('/auth/getLoginUrl').get(authController.getLoginUrl);
+openRouter.route('/auth/processSsoRedirect').post(authController.processSsoRedirect);
+authRouter.route('/auth/getLogoutUrl').get(authController.getLogoutUrl);
```

Comment:
Should the URLs be split up by dashes instead of camelCase? Not sure on the latest TFM conventions.

Reply:
yes, we need dashes

Reply:
changed to dashes